src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ReaderServiceImplTest extends Assert {
+    private static final String NONEXISTENT_FILE = "src/test/resources/notExist.csv";
+    private static final String VALID_FILE = "src/test/resources/test.csv";
+    private static final String HEADER = "type,fruit,quantity";
+    private static final String REGULAR_LINE = "b,banana,20";
+    private static ReaderService reader;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @BeforeClass
+    public static void beforeClass() {
+        reader = new ReaderServiceImpl();
+    }
+
+    @Test
+    public void read_nonExistingFile_notOk() {
+        thrown.expect(RuntimeException.class);
+        thrown.expectMessage("Can't get data from file");
```

Comment:
you can make it more informative by adding name of file or path)

Reply:
I agree, it is better to write with which file the error occurred)